docs/examples/plot_osf.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""Downloading data from the OSF
+=============================
+
```

Comment:
Generally, the way you split the information makes a lot of sense. In my opinion a few lines of intro on OSF and then pointing to the note would be beneficial even here. It is information repetition, but it would make this tutorial almost stand-alone